lib/stripe/products.rb

```diff
@@ -22,6 +23,7 @@ module Products
       validates :type, inclusion: { in: %w(service good) }
       validates :caption, :description, :shippable, :url, absence: true, unless: :good?
       validates :statement_descriptor, absence: true, unless: :service?
+      validates :unit_label, absence: true, unless: :service?
```

Comment:
what is this validation based on? I'm looking at https://stripe.com/docs/api/products/object#product_object-unit_label and it doesn't seem to mention this restriction.

> A label that represents units of this product in Stripe and on customers’ receipts and invoices. When set, this will be included in associated invoice line item descriptions.

Reply:
Hard to say at this point... maybe I have misinterpreted [this description](https://stripe.com/docs/billing/migration/products-and-plans#product) when I originally wrote this last year?

Regardless, getting rid of it :)